src/installers.ts

```diff
@@ -459,6 +474,123 @@ export const installArchiveOnlyMod: VortexWrappedInstallFunc = (
   return Promise.resolve({ instructions });
 };
 
+export const testForJsonMod: VortexWrappedTestSupportedFunc = (
+  _api: VortexAPI,
+  log: VortexLogFunc,
+  files: string[],
+  gameId: string,
+): Promise<VortexTestResult> => {
+  // Make sure we're able to support this mod.
+  const correctGame = gameId === GAME_ID;
+  log("info", "Checking JSON files for game: ", gameId);
+  if (!correctGame) {
+    // Not in game mode?
+    return Promise.resolve({
+      supported: false,
+      requiredFiles: [],
+    });
+  }
+  let supported: boolean;
+  const filtered = files.filter(
+    (file: string) => path.extname(file).toLowerCase() === JSON_FILE_EXT,
+  );
+  if (filtered.length === 0) {
+    log("info", "No JSON files");
+    return Promise.resolve({
+      supported: false,
+      requiredFiles: [],
+    });
+  }
+
+  // just make sure we don't somehow have a CET mod that got here
+  const cetModJson = filtered.filter((file: string) =>
+    path.dirname(file).toLowerCase().includes(CET_MOD_CANONICAL_PATH_PREFIX),
+  );
+  if (cetModJson.length !== 0) {
+    log("error", "We somehow got a CET mod in the JSON check");
+    return Promise.resolve({
+      supported: false,
+      requiredFiles: [],
+    });
+  }
+  let proper = true;
+  // check for options.json in the file list
```

Comment:
What’s the significance of the options.json file? Can it be captured in identifier naming so that it’s clear why we’re doing something special with it?

Reply:
`options.json` is a special case because the game ships with two different files with the same name.  One goes in `r6\config\settings` and the other in `r6\config\settings]platform\pc`